module/VuFind/src/VuFind/Connection/ExternalVuFind.php

```diff
@@ -42,10 +42,10 @@
  * @link     https://vufind.org
  */
 class ExternalVuFind implements
-    \VuFindHttp\HttpServiceAwareInterface,
+    \VuFind\Http\CachingDownloaderAwareInterface,
```

Comment:
You actually don't need the AwareInterface anymore -- that will cause the downloader to be auto-injected, which is not needed since it's now going in through the constructor. It still makes sense to use the Trait for convenience, though.

Reply:
Didn't realize about the auto-injection -- good call, fixed.